### PR TITLE
Signup

### PR DESCRIPTION
**Pull request by alxdarksage**

### CODE REVIEW

File: app/org/sagebionetworks/bridge/redis/AbstractJedisTemplate.java [hunk 1]
```diff
@@ -17,8 +17,12 @@
         String host = config.getProperty("redis.host");
         int port = config.getPropertyAsInt("redis.port");
         int timeout = config.getPropertyAsInt("redis.timeout");
-        String password = config.getProperty("redis.password");
-        JEDIS_POOL = new JedisPool(poolConfig, host, port, timeout, password);
+        if (config.isLocal()) { // No password locally.
```
Comment (eric-wu) [inline app/org/sagebionetworks/bridge/redis/AbstractJedisTemplate.java:20]: Nice!  I have also found the constructor.  I was testing (password == null) but I like yours better.

File: app/models/SignUp.java [hunk 2]
```diff
@@ -6,25 +6,32 @@
```
Comment (alxdarksage) [inline app/models/SignUp.java:6]: Controller or service layer object? Can it go into service API?

File: public/app/scripts/services/signup-service.js [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+    var signUpService = {
+        open: function(lr) {
+            modalInstance = $modal.open({
+                templateUrl: 'views/dialogs/signUp.html',
+                controller: ModalInstanceController,
+                size: 'lg',
+                windowClass: 'lg'
+            });
+        },
+        close: function() {
```
Comment (alxdarksage) [inline public/app/scripts/services/signup-service.js:41]: Is this ever called though? You only close a dialog from within the dialog itself...

File: test/webdriver/tests/BaseIntegrationTest.java [hunk 4]
```diff
@@ -34,7 +34,7 @@ public void invoke(TestBrowser browser) throws Throwable{
     }
     
     protected void call(Callback<TestBrowser> callback) {
-        running(testServer(3333, fakeApplication(inMemoryDatabase())), TestConstants.PHANTOMJS_DRIVER, wrapAndConfigureDriver(callback));
+        running(testServer(3333, fakeApplication(inMemoryDatabase())), TestConstants.FIREFOX_DRIVER, wrapAndConfigureDriver(callback));
```
Comment (alxdarksage) [inline test/webdriver/tests/BaseIntegrationTest.java:37]: AAAARGH! Change this

**Pull request merged**